packages/vscode-extension/src/commonlib/log.ts

```diff
@@ -70,12 +86,17 @@ export class VsCodeLogProvider implements LogProvider {
   /**
    * @Sample [2021-03-15T03:41:04.961Z] [Info] - [Extension] Initialize successfully.
    */
-  async log(logLevel: LogLevel, message: string): Promise<boolean> {
+  async log(logLevel: LogLevel, message: string, logToFile?: boolean): Promise<boolean> {
     try {
       if (logLevel < LogLevel.Info) return true;
       if (logLevel >= LogLevel.Warning) this.outputChannel.show();
       const dateString = new Date().toJSON();
-      this.outputChannel.appendLine(`[${dateString}] [${LogLevel[logLevel]}] - ${message}`);
+      const formattedMessage = `[${dateString}] [${LogLevel[logLevel]}] - ${message}`;
+      if (logToFile) {
```

Comment:
Does this mean the validation result will be logged into file and doesn't display in output channel anymore?